scripts/kubestellar-snapshot.sh

```diff
@@ -21,7 +21,7 @@ set -e
 # Global variables
 TIMESTAMP="$(date +%F_%T)"
 TMPFOLDER="$(mktemp -d -p . "kubestellar-snapshot-XXXX")"
-trap 'rm -rf "$TMPFOLDER"' EXIT
+trap on_exit EXIT
```

Comment:
I would feel more comfortable if `on_exit` were defined by now.

Reply:
I tested commit d28c4c509b83e9619f94ce6ec76fdda2428121c1, and got no `kubestellar-snapshot.tar.gz` file.

Reply:
Oh, right.

Reply:
I see the latest reordering.